sqlglot/expressions.py

```diff
@@ -1033,6 +1033,66 @@ class Constraint(Expression):
 class Delete(Expression):
     arg_types = {"with": False, "this": False, "using": False, "where": False, "returning": False}
 
+    def from_(self, expression, dialect=None, copy=True, **opts) -> Delete:
```

Comment:
add types. also need to add a helper method for delete like there is for select

Reply:
oh there already is one?

Reply:
I've copied those method signatures from Select, but I can add the types there, too.

Select helper method is Select.select, while the delete helper method is on the module level.

Should the delete helpher method be move from the module to Delete.delete as well? 

Currently, the delete helpher methods forces you to add a where part, I'll probably make that optional.

Reply:
If we can easily add types for these methods, that'd be ideal. Otherwise, we can defer for later, given that our methods were already untyped to begin with here.

> Select helper method is Select.select, while the delete helper method is on the module level.

We have both `exp.Select.select` and `exp.select`, which are two different callables. I think the original suggestion was to replicate this API for the `delete` functionality, i.e. have both `exp.Delete.delete` and `exp.delete`.